### ISSUE TITLE
paren mismatch issue (improvement)

### ISSUE THREAD

**Issue by pan64** [koalaman/shellcheck#2062]
#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash
component_name="${component//\//_)"

anything="asdfasd"
#### Here's what shellcheck currently says:

Line 2:
component_name="${component//\//_)"
                ^-- SC1009: The mentioned syntax error was in this parameter expansion.
 
Line 4:
anything="asdfasd"
                 ^-- SC1073: Couldn't parse this double quoted string. Fix to allow more checks.
                  ^-- SC1072: Expected end of double quoted string. Fix any mentioned problems and try again.

#### Here's what I wanted or expected to see:

would be better to detect the incorrect closing bracket in line 2
